src/test/java/com/microsoft/sqlserver/jdbc/bvt/bvtTest.java

```diff
@@ -1,678 +1,587 @@
+    private static DBPreparedStatement pstmt = null;
+    private static DBConnection conn = null;
+    private static DBStatement stmt = null;
+
+    ///////////////////////////////////////////////////////////////////
+    //// Connect to specified server and close the connection
+    /////////////////////////////////////////////////////////////////////
+    @Test
+    @DisplayName("test connection")
+    public void testConnection() throws SQLException {
+        try {
+            conn = new DBConnection(connectionString);
+            conn.close();
+        }
+        finally {
+            terminateVariation();
+        }
+    }
+
+    /////////////////////////////////////////////////////////////////////
+    //// Verify isClosed()
+    /////////////////////////////////////////////////////////////////////
+    @Test
+    public void testConnectionIsClosed() throws SQLException {
+        try {
+            conn = new DBConnection(connectionString);
+            assertTrue("BVT connection should not be closed", !conn.isClosed());
+            conn.close();
+            assertTrue("BVT connection should not be open", conn.isClosed());
+        }
+        finally {
+            terminateVariation();
+        }
+    }
+
+    /////////////////////////////////////////////////////////////////////
+    //// Verify Driver Name and Version from MetaData
+    /////////////////////////////////////////////////////////////////////
+    @Test
+    public void testDriverNameAndDriverVersion() throws SQLException {
+        try {
+            conn = new DBConnection(connectionString);
+            DatabaseMetaData metaData = conn.getMetaData();
+            Pattern p = Pattern.compile(driverNamePattern);
+            Matcher m = p.matcher(metaData.getDriverName());
+            assertTrue("Driver name is not a correct format! ", m.find());
+            String[] parts = metaData.getDriverVersion().split("\\.");
+            if (parts.length != 4)
+                assertTrue("Driver version number should be four parts! ", true);
+        }
+        finally {
+            terminateVariation();
+        }
+    }
+
+    ///////////////////////////////////////////////////////////////////
+    // Create a statement, call close
+    ///////////////////////////////////////////////////////////////////
+    @Test
+    public void testCreateStatement() throws SQLException {
+
+        try {
+            conn = new DBConnection(connectionString);
+            stmt = conn.createStatement();
+            String query = "SELECT * FROM " + table1.getEscapedTableName() + ";";
+            rs = stmt.executeQuery(query);
+            rs.verify(table1);
+        }
+        finally {
+
+            terminateVariation();
+        }
+    }
 
     ///////////////////////////////////////////////////////////////////
     // Create a statement with a query timeout
     // ResultSet.Type_forward_only,
     // ResultSet.CONCUR_READ_ONLY, executeQuery
```

Comment:
update the comments as this test doesn't use ResultSet. Also review other comments too.